app/models/ems_azure.rb

```diff
@@ -29,17 +30,16 @@ def connect(options = {})
     self.class.raw_connect(clientid, clientkey, tenant_id)
   end
 
-  def verify_credentials(_auth_type = nil, _options = {})
-    # TODO
-    true
-  end
-
-  def tenant_id=(tenant_id)
-    self.uid_ems = tenant_id
-    save
-  end
+  def verify_credentials(*)
+    begin
+      connect
```

Comment:
@Fryguy @blomquisg:
Multiple types of authentication are not supported in Azure, unlike e.g. Openstack which has default and AMQP. 
I'm wondering how and why the options hash would be used for Azure? I assume it would be configured in a script/console but I don't understand why? I'm sure there is a reason, I am just unaware of it.